### PR TITLE
[one-cmds] Add infer option to onecc

### PR DESCRIPTION
**Pull request by yunjayh**
This commit adds infer option to onecc.
Add, it contains test cases of onecc.

Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

### CODE REVIEW

**Comment by yunjayh** [comment]
```console
$ ./runtestall.sh 
onecc_027.test SUCCESS
ALL TESTS PASSED!
```

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: compiler/one-cmds/onecc [hunk 1]
```diff
@@ -43,6 +43,7 @@ subtool_list = {
     'backend': {
         'codegen': 'Code generation tool',
         'profile': 'Profile backend model file',
+        'infer': 'Infer Backend model file'
```
Comment (glistening) [inline compiler/one-cmds/onecc:46]:
```suggestion
        'infer': 'Infer backend model file'
```

**Comment by glistening** [comment]
@yunjayh Thank you for update. 

Reviewers, I just leave comment for consistent style. I am not familiar with the related code. Please review and merge at your disposal without my further approval.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@yunjayh , in anotehr PR, please add negative test to check mutual exclusiveness is working as expected.

**Pull request merged**